packages/website/vite.config.ts

```diff
@@ -4,7 +4,8 @@ export default defineConfig(({ mode }) => {
   if (mode === 'development') {
     return {
       optimizeDeps: {
-        include: ['@uidotdev/usehooks'],
+        // FIXME this causes an error with `--with-ssr`
+        // include: ['@uidotdev/usehooks'],
```

Comment:
I want to make website work in development in both with and without `--with-ssr`.